allensdk/model/biophysical/utils.py

```diff
@@ -310,6 +315,10 @@ def nearest_neuron_sampling_rate(hz, target_hz=40000):
 
 
 class AllActiveUtils(Utils):
+    
+    def __init__(self, description,axon_type='reconstructed_axon'):                    
```

Comment:
small style point: put a space after this comma. See the [google python style guide](https://google.github.io/styleguide/pyguide.html) for more details like this. Also, consider running [`pylint`](https://www.pylint.org/) to detect these sorts of things automatically.